src/test/Rules/CreateRule.test.tsx

```diff
@@ -214,19 +202,19 @@ describe('<CreateRule />', () => {
     expect(createButton).toBeInTheDocument();
     expect(createButton).toBeVisible();
 
-    userEvent.type(nameInput, mockRule.name);
-    userEvent.type(descriptionInput, mockRule.description);
-    userEvent.type(matchExpressionInput, escapeKeyboardInput(mockRule.matchExpression));
-    userEvent.selectOptions(templateSelect, [screen.getByText('Profiling')]);
-    userEvent.type(maxSizeInput, `${mockRule.maxSizeBytes}`);
-    userEvent.type(maxAgeInput, `${mockRule.maxAgeSeconds}`);
-    userEvent.type(archivalPeriodInput, `${mockRule.archivalPeriodSeconds}`);
-    userEvent.type(preservedArchivesInput, `${mockRule.preservedArchives}`);
-    userEvent.type(initialDelayInput, `${mockRule.initialDelaySeconds}`);
+    await user.type(nameInput, mockRule.name);
```

Comment:
Maybe here do 
`await Promise.all([user.type(..,..), user2, user3, etc...])` if this would work?
Doesn't really matter but may save some milliseconds on throughput when order doesn't matter. (Some of the order does matter though, (selectOptions), so probably just put that synchronously at the front).

Reply:
Oh niiceee good idea! I will try that.

Reply:
Unfortunately, using `Promise.all` seems to causes some of microtasks on queue got ignored. This causes the Create button to still be disabled. Maybe we will figure another way later.